kmod/src/Makefile.kernelcompat

```diff
@@ -40,8 +40,15 @@ endif
 #
 # Added user_ns argument to posix_acl_valid
 #
-ifneq (,$(shell grep 'posix_acl_valid.*user_ns,' include/linux/posix_acl.h))
-ccflags-y += -DKC_POSIX_ACL_VALID_USER_NS
+ifneq (,$(shell grep 'posix_acl_valid.*user_namespace' include/linux/posix_acl.h))
```

Comment:
It's only the definition that changed, the code still calls it user_ns so let's not change the KC define.  Makes it a tiny patch on its own.